modules/validate.nf

```diff
@@ -1,17 +1,18 @@
 // validate submission results
-process VALIDATE {
+process VALIDATE {    
     secret "SYNAPSE_AUTH_TOKEN"
     container "python:3.12.0rc1"
 
     input:
     tuple val(submission_id), path(predictions)
     val ready
+    val validation_script
 
     output:
     tuple val(submission_id), path(predictions), stdout, path("results.json")
 
     script:
     """
-    validate.py '${predictions}'
+    ${validation_script} '${predictions}'
```

Comment:
Ah, I see.  I think we should support different containers, but limit the inputs and outputs of that container.